docs/UpAndRunning.md

```diff
@@ -0,0 +1,19 @@
+# RapydBot - Up and Running
+
+Here you will find everything you need to know to be able to test RapydBot in your own local environment.
+
+## Prerequisites
+<hr>
+Before you begin, check that you have the following programs installed on your computer:
+<br>
+<br>
+
+- [Git](https://git-scm.com/downloads)
+- [Ngrok](https://ngrok.com/download)
+- [Node JS - Stable Version](https://nodejs.org/es/download/)
+- [Yarn](https://yarnpkg.com/)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+Make sure to follow the instructions provided for the installation of the above programs based on your operating system.
+
+In this guide we will use **Yarn** but feel free to use your favorite Node package manager.
```

Comment:
@Ktoxcon 

Let's add an `Installation` header with the instructions on how to boot `bot`, `user` & `wallet` 💪